parse/parser.go

```diff
@@ -59,6 +61,8 @@ type precedence int
 const (
 	_ precedence = iota
 	LOWEST
+	LOR         // ||
+	LAND        // &&
```

Comment:
I think LOR and LAND have same precedence. plz check with test case which contain both `||` and `&&`

for example
```
true || true && true // should ((true || true) && true)
```
```
true && true || true // should ((true && true) || true)
```

Reply:
complete > <

Reply:
my mistake 

https://golang.org/src/go/token/token.go